prediction_prophet/deployment/models.py

```diff
@@ -58,12 +58,13 @@ def calculate_bet_amount(self, answer: bool, market: AgentMarket) -> BetAmount:
         elif isinstance(market, OmenAgentMarket):
             # On Omen, we bet between 0.5 and 1 xDai based on the market's current probability
             # (we bet more if market is less saturated and we can win more).
+            market_liquidity = market.get_liquidity_in_xdai()
             amount = Decimal(stretch_bet_between(
```

Comment:
Just thinking, based on Martin's idea of allowing the user to bet based on the difference between their predicted `p_yes` and the  market's current `p_yes`, we'd need to make that information available here. I've made an issue for it here https://github.com/gnosis/prediction-market-agent-tooling/issues/161.

We cooould in the meantime roughly approximate it here by betting more if `answer` != market.p_yes > 0.5`, but will leave it up to you :)